packages/drivers/odsp-driver/src/graph.ts

```diff
@@ -90,14 +94,16 @@ export async function graphFetch(
     requestInit?: RequestInit,
     retryPolicy?: RetryPolicy<Response>,
     timeoutMs = 0,
+    usePPE?: boolean,
 ): Promise<Response> {
     const getToken = async (options: TokenFetchOptions) =>
         getShareLinkToken(
             options,
             SharingLinkScopeFor.nonFileDefaultUrl,
             siteUrl,
         );
-    const url = graphUrl.startsWith("http") ? graphUrl : `https://graph.microsoft.com/v1.0/${graphUrl}`;
+    const url = graphUrl.startsWith("http") ? graphUrl :
```

Comment:
can this logic move up the stack?
Function should either receive (non-optimal, proper) URL, or it should receive no URL but boolean indicating what environment to target. Otherwise contract is really broken - I can pass URL, but also pass usePPE, but it is ignored.
If absolutely not possible, as a last resort these two args need to be merged into one string | boolean (or type with same idea).